docs/guide/acquiring_data/hek.rst

```diff
@@ -15,7 +15,7 @@ support from the European Space Agency Summer of Code in Space
 SunPy's HEK module is in sunpy.net.  It can be imported into your
 session as follows:
 
-    >>> from sunpy.net import hek
+    >>> from sunpy.net import attrs as a
     >>> client = hek.HEKClient()  # doctest: +REMOTE_DATA
```

Comment:
Do we want to update this with fido instead now?

Reply:
Agreed